upload.sh

```diff
@@ -28,14 +30,17 @@ python3 get-pip.py
 pip install boto3
 
 # get the paths & filenames of the files to upload
-ISOPATH="$(find /artifacts -name "*.iso")"
-ISO="$CHANNEL/$(basename "$ISOPATH")"
-ISOTAG="$(basename "$ISOPATH" .iso)"
-SHAPATH="$(find /artifacts -name "*.sha256.txt")"
-SHASUM="$CHANNEL/$ISOTAG.sha256.txt"
-MD5PATH="$(find /artifacts -name "*.md5.txt")"
-MD5="$CHANNEL/$ISOTAG.md5.txt"
-
-python3 upload.py "$KEY" "$SECRET" "$ENDPOINT" "$BUCKET" "$ISOPATH" "$ISO"
-python3 upload.py "$KEY" "$SECRET" "$ENDPOINT" "$BUCKET" "$SHAPATH" "$SHASUM"
-python3 upload.py "$KEY" "$SECRET" "$ENDPOINT" "$BUCKET" "$MD5PATH" "$MD5"
+ISOPATHS="$(find /builds -name "*.iso")"
```

Comment:
It seems wrong that this is `/builds`

I don't think `builds` is a folder at the root of the filesystem at this point?